.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -50,7 +50,7 @@ task nextstrain_workflow {
     export S3_FILESTEM="~{s3_filestem}"
     export GROUP_NAME="~{group_name}"
     # We aren't supposed to serialize Map inputs on the command line.
-    export TEMPLATE_ARGS_FILE="~{write_json(template_args)}"
+    export TEMPLATE_ARGS_FILE="~{write_lines([template_args])}"
```

Comment:
We use a slightly different function to write strings instead of WDL-native structs.